Sky Atlas/Sky Atlas.html

```diff
@@ -17958,7 +18258,10 @@ <h1>Annotations</h1>
           </td>
           <td>Delegated Voting Power - Element Annotation</td>
           <td>Annotation</td>
-          <td>The element “delegated voting power” refers to the ability of SKY holders and Sealing users to entrust their voting power to a Delegate Contract, allowing the Aligned Delegate controlling that contract to cast votes in the Sky Governance process on their behalf. The SKY holder or Sealing user always maintains full ownership of their tokens.</td>
+          <td>The element “delegated voting power” refers to the ability of SKY holders and Staking users to
+            entrust their voting power to a Delegate Contract, allowing the Aligned Delegate controlling that
+            contract to cast votes in the Sky Governance process on their behalf. The SKY or Staking user always
+            maintains full ownership of their tokens.</td>
```

Comment:
Don't add line breaks

```suggestion
          <td>The element “delegated voting power” refers to the ability of SKY holders and Staking users to  entrust their voting power to a Delegate Contract, allowing the Aligned Delegate controlling that contract to cast votes in the Sky Governance process on their behalf. The SKY or Staking user always maintains full ownership of their tokens.</td>
```